service/routes.py

```diff
@@ -254,3 +254,43 @@ def update_shopcart_item(shopcart_id, item_id):
         return request_validation_error(e)
     except Exception as e:
         return internal_server_error(e)
+
+
+######################################################################
+# DELETE AN ITEM
+######################################################################
+@app.route("/shopcarts/<int:shopcart_id>/items/<int:item_id>", methods=["DELETE"])
+def delete_items(shopcart_id, item_id):
+    """
+    Delete an item
+
+    This endpoint will delete an item based the id specified in the path
+    """
+   
+    app.logger.info(f"Request to delete item with id='{item_id}' in shopcart with id='{shopcart_id}'.")
+     # See if the shopcart exists and abort if it doesn't
+    shopcart = Shopcart.get_by_id(shopcart_id)
+    if not shopcart:
+        abort(
```

Comment:
I have no good answer of why line 274 in `routes.py` is still count as a missing line given the test case you wrote `test_delete_item_shopcart_not_found()` 🤔

Reply:
However, you may refer to this [commit](https://github.com/CSCI-GA-2820-SU23-001/shopcarts/pull/47/commits/e0f0fc7ab2dafd3086940cd23705b1c17cd9b047) to add a test case specifically for `shopcart=None`, with this line 274 in `routes.py` would no longer be counted as a line without test!

Reply:
Let me try this approach, thanks for your suggestion!